sdk/loadtestservice/Azure.Developer.LoadTesting/SampleCodes/Program.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.IO;
+using System.Net.Http.Headers;
+using System.Net.Http;
+using Azure;
+using Azure.Core;
+using Azure.Developer.LoadTesting;
+using Azure.Identity;
+using System.IO.Pipes;
+using System.Text.Json.Serialization;
+using Newtonsoft.Json;
+using System.Net;
+using System.Net.Mime;
+
+namespace SampleCodes
+{
+    internal class Program
```

Comment:
Is there something here that isn't already covered by the samples in `/samples`?

Reply:
@christothes earlier I was writing Samples as a different project itself, shifted to /samples folder as suggested in the docs.